modules/afsocket/tests/test-transport-mapper-inet.c

```diff
@@ -31,6 +31,13 @@
 
 #include <unistd.h>
 
+guint SCS_TCP;
+guint SCS_TCP6;
+guint SCS_UDP;
+guint SCS_UDP6;
+guint SCS_NETWORK;
+guint SCS_SYSLOG;
+
 static inline void
```

Comment:
these are pretty ugly again. if we followed the same pattern (e.g. pass a const char * to the function that needs the stats identifier), we could get rid of these variables.